nshlib/Kconfig

```diff
@@ -728,16 +728,6 @@ config NSH_DISABLE_LOOPS
 
 endif # !NSH_DISABLESCRIPT
 
-config NSH_MMCSDMINOR
```

Comment:
Isn't better to remove the other one?  This one provides a better comment, IMO.

Reply:
> Isn't better to remove the other one? This one provides a better comment, IMO.

No, that place contain other MMCSD related definition. So, I replace the description at that location with this one. Please review again.

Reply:
I see, you're right.  LGTM!